src/main/java/com/thealgorithms/others/FloydTriangle.java

```diff
@@ -1,22 +1,28 @@
 package com.thealgorithms.others;
 
-import java.util.Scanner;
-
 final class FloydTriangle {
     private FloydTriangle() {
     }
 
-    public static void main(String[] args) {
-        Scanner sc = new Scanner(System.in);
-        System.out.println("Enter the number of rows which you want in your Floyd Triangle: ");
-        int r = sc.nextInt();
-        int n = 0;
-        sc.close();
-        for (int i = 0; i < r; i++) {
+    /**
+     * Generates a Floyd Triangle with the specified number of rows.
+     *
+     * @param rows The number of rows in the triangle.
+     * @return A string representing the Floyd Triangle.
+     */
+    public static String generateFloydTriangle(int rows) {
```

Comment:
Could you make it output jagged array instead of a string? To keep presentation separate from the actual triangle

Reply:
Also thought about this, because it is better. But left initial implementation. Fixed.